packages/test/test-end-to-end-tests/src/test/summarization/summarizeRestart.spec.ts

```diff
@@ -43,8 +43,13 @@ describeCompat(
 			return provider.makeTestContainer(testContainerConfig);
 		};
 
-		beforeEach("setup", async () => {
+		beforeEach("setup", async function () {
 			provider = getTestObjectProvider({ syncSummarizer: true });
+			// Run these tests only for Local and ODSP drivers. Ideally we only need to run tests with local server since we are only testing the summarizer's logic independednt of the service,
+			// but keeping ODSP in the mix to also test code with single commit summaries. (ODSP has single commit summaries enabled by default)
+			if (["local", "odsp"].includes(provider.driver.type)) {
```

Comment:
This seems like the opposite of what you intended?

Reply:
Yup (_insert_facepalm_emoji_here)